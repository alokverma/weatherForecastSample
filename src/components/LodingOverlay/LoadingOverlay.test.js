import React from 'react';
import {shallow} from 'enzyme';
import LoadingOverlay from './index'

describe('LoadingOverlay',()=>{
    it('should render list LoadingOverlay component',()=>{
        const component = shallow(<LoadingOverlay />)
        expect(component).toMatchSnapshot()

    });
});