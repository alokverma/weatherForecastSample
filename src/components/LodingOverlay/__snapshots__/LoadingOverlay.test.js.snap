// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoadingOverlay should render list LoadingOverlay component 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "rgba(0,0,0,0.7)",
      "bottom": 0,
      "flex": 1,
      "justifyContent": "center",
      "left": 0,
      "position": "absolute",
      "right": 0,
      "top": 0,
    }
  }
>
  <View>
    <Loading
      size="large"
    />
  </View>
</View>
`;
